.buildkite/pipelines/scheduled/coverage/coverage_linux64.yml

```
@@ -29,6 +29,9 @@ steps:
       git config --global init.defaultBranch master
 
       echo "--- Run Julia tests in parallel with code coverage enabled"
+      export JULIA_NUM_THREADS=1
```

```suggestion
      unset JULIA_NUM_THREADS
```

But why is this set at all? It is not a good base configuration.

---

My memory is hazy, but if I recall correctly, we didn't use to have this set, and then some job started 256 Julia threads (because the underlying machine had 128 physical cores, 256 CPU threads) and clobbered the whole machine, so we set this variable.

---

We also set `JULIA_CPU_THREADS` to `16`. Maybe that is sufficient, and we can remove `JULIA_NUM_THREADS` from the default configuration?

---

@staticfloat will probably remember why we set `export JULIA_NUM_THREADS=16` on all the Buildkite agents.

Maybe we should set the following configuration on all of the Buildkite agents, just to be safe:
```
export JULIA_CPU_THREADS=16
export JULIA_NUM_THREADS=1
```